### ISSUE TITLE
Clarify usage and registration of new taggs for future extensions

### ISSUE THREAD

**Issue by ghobona** [opengeospatial/geotiff#111]
The OGC-NA maintains a [register of GeoTIFF tags](http://www.opengis.net/def/geotiff-tag). 

In the future it may be necessary for communities that use COG to register tags for their specialist use cases.

Such tags could then be organised into groups that deal with the needs of a particular domain.

See this [OWS-9 engineering report](https://portal.ogc.org/files/?artifact_id=51889) for some examples of metadata for specialist use cases.

So the COG spec could include a mention of the GeoTIFF Tag Register.

**Comment by joanma747** [comment]
I agree in principle but I also have may objections. The "GeoTIFF tags" are defined in the GeoTIFF standard and IMHO cannot be extended without editing the standard. What is possible for the communities is to define new "TIFF tags" outside the GeoTIFF specification . But, defining TIFF tags is complicated as there is no common registry for that (extraofficial one here: https://www.awaresystems.be/imaging/tiff/tifftags.html). An alternative for communities could be to define new GeoTIFF keys inside the current GeoTIFF tags by extending GeoTIFF standard to other domains. This means that we might need a registry of GeoTIFF keys too that is extensible. Do we have a registry for geotiff keys?.

For the moment I have added the following text to the standard:
"The OGC-NA maintains a register of TIFF tags relevant for the geospatial community (most of them defined in the GeoTIFF standard) with the URL http://www.opengis.net/def/geotiff-tag. The COG standard does not define any COG specific TIFF tags or GeoTIFF keys. In the future it may be necessary for the communities that use COG to register tags for their specialist use cases. It is also possible that they reuse the existing TIFF tags to define new GeoTIFF keys. Such tags or keys could then be organized into groups that deal with the needs of a particular domain."
